clinica/pipelines/pet_surface/pet_surface_utils.py

```diff
@@ -872,7 +914,7 @@ def get_wf(subject_id,
                            name='vol2vol_mask')
 
     if 'SPMSTANDALONE_HOME' in os.environ:
-        tpmnii = os.path.join(str(os.getenv("SPM_HOME")), 'spm12_mcr/spm12/spm12/tpm/TPM.nii')
```

Comment:
If `SPMSTANDALONE_HOME` exists, `SPM_HOME` does not necessarily exists. + The path with `spm12/spm12` was not correct

Reply:
On the `dev` branch of Clinica, there is now a `get_tpm()` function (see [spm.py](https://github.com/aramis-lab/clinica/blob/dev/clinica/utils/spm.py)) to extract the tissue probability map. You should use/modify this to avoid duplicated lines.

Besides, I am currently refactoring some code while I'm finding a way to run ReconAll for regression tests. Anyway, I have noticed this:
![GetSpmHome](https://user-images.githubusercontent.com/24808663/65252414-386ead00-daf9-11e9-9569-b02ab705167c.png)

I extracted the lines into a single `get_spm_home()` function but the latter assumes that `SPM_HOME` exists based on lines. 

This would mean that there is a possible bug in `atlas.py` when using SPM Standalone....

Reply:
The path was right for Linux system indeed. See below.

For `get_atlas_labels()`: the lines

```python 
            if "SPMSTANDALONE_HOME" in os.environ:
                if platform.system() == 'Darwin':
                    SPM_HOME = os.environ['SPMSTANDALONE_HOME'] + "/spm12.app/Contents/MacOS/spm12_mcr"
``` 
make no sense to me. Here is the content of `$SPMSTANDALONE_HOME/spm12.app/Contents/MacOS/` : 
`applauncher  prelaunch    spm12.ctf    spm12_maci64`

Reply:
In that case, I would advice to create an issue regarding this misleading point and fix this once for all.

Would that suit you?